ateam_kenobi/src/core/motion/motion_controller.cpp

```diff
@@ -195,10 +199,23 @@ ateam_msgs::msg::RobotMotionCommand MotionController::get_command(
 
 void MotionController::reset()
 {
-  // TODO(anon): handle pid gains better
-  this->x_controller.initialize(2.8, 0.0, 0.002, 0.3, -0.3, true);
-  this->y_controller.initialize(2.8, 0.0, 0.002, 0.15, -0.15, true);
-  this->t_controller.initialize(2.5, 0.0, 0.0, 0.5, -0.5, true);
+  const auto u_max = std::numeric_limits<double>::infinity();
```

Comment:
Would it be better to set these to like the max velocity or something just to make sure the controller doesn't go completely crazy?